testproject/Assets/Tests/Runtime/testproject.runtimetests.asmdef

```diff
@@ -1,11 +1,24 @@
 {
     "name": "TestProject.RuntimeTests",
+    "rootNamespace": "",
     "references": [
-        "Unity.Multiplayer.MLAPI.Runtime"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Multiplayer.MLAPI.Runtime",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner",
+        "Unity.Multiplayer.MLAPI.RuntimeTests",
+        "TestProject.ManualTests"
```

Comment:
changes could be limited to only 2 added lines and the rest of the diff should not be here (might require manual text edit to prevent Unity's auto gen):

```suggestion
        "Unity.Multiplayer.MLAPI.Runtime",
        "Unity.Multiplayer.MLAPI.RuntimeTests",
        "TestProject.ManualTests"
```

also, `Testproject.RuntimeTests` depending on `TestProject.ManualTest` + `MLAPI.RuntimeTests` doesn't sound right to me.
we started to go a little bit crazy in dependencies I think :D
is this because we're not organizing code better or are these dependencies inevitable?

Reply:
Did you test this change before you suggested it?

Reply:
yes.

Reply:
here you go:
```json
{
    "name": "TestProject.RuntimeTests",
    "references": [
        "Unity.Multiplayer.MLAPI.Runtime",
        "Unity.Multiplayer.MLAPI.RuntimeTests",
        "TestProject.ManualTests"
    ],
    "optionalUnityReferences": [
        "TestAssemblies"
    ]
}
```

Reply:
Nice suggestion... 
>     "optionalUnityReferences": [
>         "TestAssemblies"
>     ]

Assures the test assemblies will be loaded:
        "UnityEngine.TestRunner",
        "UnityEditor.TestRunner",